app/components/file-details/api-scan/captured-apis/index.ts

```diff
@@ -151,6 +170,40 @@ export default class FileDetailsApiScanCapturedApisComponent extends Component<F
     }
   });
 
+  selectAllCapturedApis = task(async (is_active: boolean) => {
+    this.showAPIListLoadingState = false;
+
+    try {
+      await this.ajax.put(this.modAllSelectedAPIsEndpoint, {
+        data: { is_active: is_active },
+        namespace: ENV.namespace_v2,
+      });
+
+      await this.fetchCapturedApis.perform(this.limit, this.offset);
+      await this.getAllAPIsSelectedStatus.perform();
```

Comment:
why do we require this call ?

Reply:
Yes we do. We need to know the status of the select all checkbox when we visit the page at any point in time.